README.md

```diff
@@ -0,0 +1,90 @@
+<!--
+Avoid using this README file for information that is maintained or published elsewhere, e.g.:
+
+* metadata.yaml > published on Charmhub
+* documentation > published on (or linked to from) Charmhub
+* detailed contribution guide > documentation or CONTRIBUTING.md
+
+Use links instead.
+-->
+
+# matrix-operator
+
+Charmhub package name: matrix-operator
+More information: https://charmhub.io/matrix-operator
+
+Describe your charm in one or two sentences.
```

Comment:
Looks like the default readme?

Reply:
Thanks, changed it and cleaned up the README a bit.